aws/redshift/queries/annual_master_metrics/csd_csp_pd_high_needs_rural.sql

```diff
@@ -94,7 +94,7 @@ join scripts sc on sc.id = us.script_id
 join users u_student on u_student.id = f.student_user_id
 join csp_csd_teachers_trained tt on tt.studio_person_id = u.studio_person_id and course = 'CS Discoveries'
 join school_stats ss on ss.school_id = tt.school_id
-where sc.id in (select script_id from course_scripts cs join courses c on c.id = cs.course_id where c.name in ('csd-2017','csd-2018', 'csd-2019'))
```

Comment:
These references to the `courses` table were not covered by any tests. Any other places to check for DB queries without test coverage?

Reply:
Good find. This makes me think about:

- Have you given a heads up to the data team (whoever that is now) about these changes? Maybe it'll break things for them aside from this one set of queries, maybe things in tableau? (Also, I think there's at least one other file in the redshift queries directory that references this table?)
- We probably need to figure out what to do with our DMS configuration: https://github.com/code-dot-org/code-dot-org/blob/5a89765b48fb1b3e15b4027f819147bedc1fd957/aws/dms/tasks.yml#L50

Some other directories I could think of to recursively grep: `bin/cron`, `aws`, and `cookbooks`.  I didn't find anything, though.

Reply:
Thanks for catching this, @uponthesun . aws/dms/tasks.yml will be addressed in https://github.com/code-dot-org/code-dot-org/pull/36086 which will be merged before the courses table is dropped.